app/src/main/java/co/nimblehq/compose/crypto/extension/BoxShadow.kt

```diff
@@ -0,0 +1,53 @@
+package co.nimblehq.compose.crypto.extension
+
+import android.graphics.BlurMaskFilter
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Paint
+import androidx.compose.ui.graphics.drawscope.drawIntoCanvas
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+fun Modifier.boxShadow(color: Color = Color.Black,
+                               borderRadius: Dp = 0.dp,
+                               blurRadius: Dp = 0.dp,
+                               offsetY: Dp = 0.dp,
+                               offsetX: Dp = 0.dp,
+                               spread: Float = 0f,
+                               modifier: Modifier = Modifier,
+) = this.then(
+    modifier.drawBehind {
+        this.drawIntoCanvas {
+            val paint = Paint()
+            val frameworkPaint = paint.asFrameworkPaint()
+            val spreadPixel = spread.dp.toPx()
+            val leftPixel = (0f - spreadPixel) + offsetX.toPx()
+            val topPixel = (0f - spreadPixel) + offsetY.toPx()
+            val rightPixel = (this.size.width + spreadPixel)
+            val bottomPixel =  (this.size.height + spreadPixel)
+
+
```

Comment:
Would you mind if remove this redundant breakline

Reply:
@hoangnguyen92dn Ok will do & I will remove the bracket